src/presenters/business/business-email-change-presenter.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Formats data ready for presenting in the `/business-details` page
+ * @module businessEmailChangePresenter
+ */
+
+const businessEmailChangePresenter = (data, yar) => {
+  return {
+    notification: yar ? yar.flash('notification')[0] : null,
```

Comment:
I don't think we need notification here becuase the email check page doesn't require a banner